ui/imodel-components-react/README.md

```diff
@@ -6,6 +6,19 @@ Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md
 
 The **@itwin/imodel-components-react** package contains React components that depend on the @itwin/core-frontend, @itwin/core-common or @itwin/core-quantity packages.
 
-## Documentation
+## Usage
+
+To get started with React components from the `@itwin/imodel-components-react` package, simply import the necessary components and utilities:
+
+```tsx
+import { BaseSolarDataProvider } from "@itwin/imodel-components-react";
+
+export const Basic: Story = {
```

Comment:
Let's avoid storybook specific snippets.

Reply:
Updated the code sample.